docs/specs/aerogear-digger-node-client/index.html

```diff
@@ -0,0 +1,137 @@
+
+    
+
+
+
+    
+
+
+    <h3> </h3>
+
+
+
+
+
+
+
+
+
+
+    
+
+
+
+
+    <section>
+        <article><h1>AeroGear Digger Node.js client</h1><p><a href="https://travis-ci.org/aerogear/digger-node"><img src="https://travis-ci.org/aerogear/digger-node.png" alt="Build Status"></a>
+<a href="http://www.apache.org/licenses/LICENSE-2.0"><img src="https://img.shields.io/:license-Apache2-blue.svg" alt="License"></a></p>
+<p>AeroGear digger node.js command line client. 
+Create Jenkins job and build your application using Jenkinsfile located in your source code.</p>
+<h2>Project Info</h2><table>
+<thead>
+<tr>
+<th></th>
+<th>Project Info</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>License:</td>
+<td>Apache License, Version 2.0</td>
+</tr>
+<tr>
+<td>Build:</td>
+<td>Node</td>
+</tr>
+<tr>
+<td>Documentation:</td>
+<td>https://github.com/aerogear/digger-jenkins</td>
+</tr>
+<tr>
+<td>Issue tracker:</td>
+<td>https://issues.jboss.org/browse/AGDIGGER</td>
+</tr>
+<tr>
+<td>Mailing lists:</td>
+<td><a href="http://aerogear-users.1116366.n5.nabble.com/">aerogear-users</a> (<a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">subscribe</a>)</td>
+</tr>
+<tr>
+<td></td>
+<td><a href="http://aerogear-dev.1069024.n5.nabble.com/">aerogear-dev</a> (<a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">subscribe</a>)</td>
+</tr>
+<tr>
+<td>IRC:</td>
+<td><a href="https://webchat.freenode.net/?channels=aerogear">#aerogear</a> channel in the <a href="http://freenode.net/">freenode</a> network.</td>
+</tr>
+</tbody>
+</table>
+<h2>Installation</h2><p>Install command line tool</p>
+<p><code>npm install -g digkins</code></p>
+<h2>Sample use case</h2><p>Login to Jenkins. Your credentials will be stored in configuration.</p>
+<pre class="prettyprint source"><code>digkins login http://myjenkins.com</code></pre><p>Create job that would use your repository as source code</p>
+<pre class="prettyprint source"><code>digkins job create my-job https://github.com/android/project master</code></pre><p>Trigger build</p>
+<pre class="prettyprint source"><code>digkins job build my-job</code></pre><h2>Supported commands</h2><pre class="prettyprint source"><code>  # Setup jenkins credentials and login into jenkins
+  digkins login &lt;url> [user] [password]
+
+  # Create jenkins job for git repository with Jenkinsfile
+  digkins job create &lt;name> [repository] [branch]
+
+  Trigger build for Jenkins job
+  digkins job build &lt;jobname>                
+
+  # Stream jenkins logs for triggered build
+  digkins log &lt;job> &lt;buildNumber>       
+
+  # Get job artifacts for specified build
+  digkins artifact &lt;job> &lt;buildNumber>   
+
+  # Generate bash completion script
+  digkins completion</code></pre><h2>Development</h2><ol>
+<li>Install node.js</li>
+<li>Checkout repository</li>
+<li>Link library to use directly from source code</li>
+</ol>
+<p><code>npm link .</code> </p></article>
+    </section>
+
+
+
+
+
+
+</div>
+
+<nav>
```

Comment:
I think the links are a bit funny here: Could we change `Global` to a heading like `Modules` and have a link underneath it?

Reply:
this is generated

Reply:
:+1: